src/main/java/lotto/constants/Rank.java

```diff
@@ -33,4 +33,11 @@ public static Rank valueOf(int countOfMatch, boolean matchBonus) {
                 .findFirst()
                 .orElse(MISS);
     }
+
+    public boolean isWinner() {
+        if (this.countOfMatch >= FIFTH.countOfMatch) {
+            return true;
+        }
+        return false;
```

Comment:
간단하게 인라인 표현 해볼수 있겠네요!

```suggestion
        return this.countOfMatch >= FIFTH.countOfMatch;
```

Reply:
넵 수정하겠습니다